src/node/chainstate.cpp

```diff
@@ -55,14 +56,14 @@ static ChainstateLoadResult CompleteChainstateInitialization(
         }
     }
 
-    if (options.check_interrupt && options.check_interrupt()) return {ChainstateLoadStatus::INTERRUPTED, {}};
+    if (chainman.m_interrupt) return {ChainstateLoadStatus::INTERRUPTED, {}};
```

Comment:
MIght this be a regression in case of the `chainman` reference being eventually removed from the `chainstate`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/28051#discussion_r1304824364

> MIght this be a regression in case of the `chainman` reference being eventually removed from the `chainstate`?

Is this referencing https://github.com/bitcoin/bitcoin/pull/28218#discussion_r1291351347? If so, I don't think I understand the connection because there isn't a Chainstate object here.

Reply:
Yeah, right. I don't think this applies here either now. Please resolve.